src/AppContext.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { defaultAuthenticatedUser } from './frontendAuthWrapper';
+
+const AppContext = React.createContext({
+  authenticatedUser: defaultAuthenticatedUser,
```

Comment:
Any downside to just naming this `user`?

Reply:
We use the prefix "authenticated" in frontend-auth, fwiw.  Mostly it was just to keep it consistent.  But... Hm.  🤔